docker/compose/docker-compose-console.yml

```diff
@@ -1,11 +1,12 @@
 version: "3.7"
 
 x-fdg-basic: &fdg-basic
-  image: ghcr.io/robotology/assistive-rehab:v2022.05.2-basic
+  image: ghcr.io/robotology/assistive-rehab:master-basic-devel
```

Comment:
We don't yield any `...-basic-devel` image via the action.

Reply:
Fixed by removing the `-devel` suffix, good point.